crates/motsu/src/context.rs

```diff
@@ -95,15 +265,156 @@ unsafe fn write_bytes32(ptr: *mut u8, bytes: Bytes32) {
     ptr::copy(bytes.as_ptr(), ptr, WORD_BYTES);
 }
 
-/// Initializes fields of contract storage and child contract storages with
-/// default values.
-pub trait DefaultStorage: StorageType {
-    /// Initializes fields of contract storage and child contract storages with
-    /// default values.
-    #[must_use]
+/// Storage for unit test's mock data.
+#[derive(Default)]
+struct MockStorage {
+    /// Address of the message sender.
+    msg_sender: Option<Address>,
+    /// Address of the contract that is currently receiving the message.
+    contract_address: Option<Address>,
+    /// Contract's address to mock data storage mapping.
+    contract_data: HashMap<Address, ContractStorage>,
+    // Output of a contract call.
+    call_output: Option<Vec<u8>>,
+    // Output length of a contract call.
+    call_output_len: Option<usize>,
+}
+
+type ContractStorage = HashMap<Bytes32, Bytes32>;
+
+/// Contract call entity, related to the contract type `ST` and the caller's
+/// account.
+pub struct ContractCall<ST: StorageType> {
+    contract: ST,
```

Comment:
```suggestion
    storage: ST,
```
Wouldn't this be a more accurate name? I find `contract` confusing, as it leads me to believe it stores [struct Contract](https://github.com/OpenZeppelin/stylus-test-helpers/pull/14/files#diff-ced01b2c555b639e21eb87ee2504510248dd67de0b0ca5d720243942b15c0fd9R325). Maybe even `instance: ST` would be a clearer name (as in "contract storage instance") :thinking:

Reply:
Why not. Will change it